internal/proposals/unified-approach-to-logging-levels.md

```diff
@@ -0,0 +1,50 @@
+# Unified approach to logging levels
+
+It must be possible to easily set a more verbose logging level of each component within the Kyma ecosystem while troubleshooting.
+
+We also have some "external" components (not implemented by the Kyma team) and we need an approach on how to change the logging level in them. For debugging reasons the level could be changed without any code and charts changes just with the in-cluster resource change.
+
+## Default severity levels for the environments
+
+To be consistent, we need to agree on what default level components will log in each environment:
+
+- PROD: `INFO` - this is a minimal severity needed for performing the audit logging,
```

Comment:
Do we need the INFO level on our prod environments? Are there any components other than Dex that are logging audit-log relevant stuff? I would reduce that to the WARN/ERROR since the logs do have their cost and we would like to reduce that.

Reply:
I was thinking about it and I decided to leave the `INFO` there and discuss it later. I also think the `WARN` level would be much better, just didn't know the exact approach to audit logging we currently perform. I'll happily change it to the `WARN` level then!